sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/windowing/DefaultTrigger.java

```diff
@@ -91,4 +91,9 @@ private boolean endOfWindowReached(Trigger.TriggerContext context) {
 
   @Override
   public void onFire(Trigger.TriggerContext context) throws Exception { }
+
+  @Override
+  public String toString() {
+    return "Repeatedly.forever(AfterWatermark.pastEndOfWindow)";
```

Comment:
Actually, what if we just omit display data for the default trigger (and leave the `toString()` implementation as-is). This is consistent with other places where we exclude defaults.